internal/models/credential/aws_spec.go

```diff
@@ -0,0 +1,70 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package credentialmodels
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1AccountCredentialTypeAwsSpec AWS credential spec.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.account.credential.type.aws.Spec
+type VmwareTanzuManageV1alpha1AccountCredentialTypeAwsSpec struct {
+
+	// AWS access key ID and secret access key.
+	AccessKey *VmwareTanzuManageV1alpha1AccountCredentialTypeAwsAccessKey `json:"accessKey,omitempty"`
+
+	// Account ID of the AWS credential.
+	AccountID string `json:"accountId,omitempty"`
+
+	// Generic credential.
+	GenericCredential string `json:"genericCredential,omitempty"`
+
+	// AWS IAM role ARN and external ID.
+	IamRole *VmwareTanzuManageV1alpha1AccountCredentialTypeAwsIAMRole `json:"iamRole,omitempty"`
+}
+
+// MarshalBinary interface implementation
```

Comment:
These are models and they are automated files and linter is not complaining.

Reply:
Strange, previously even the model files used to have this failure, and we added this period in all statements for other resource models. Anyways, since it's passing, we can let it be.